docs/api/datasets/non_geo_datasets.csv

```diff
@@ -39,7 +39,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `Million-AID`_,C,Google Earth,-,1M,51--73,,0.5--153,RGB
 `MMEarth`_,"C, S","Aster, Sentinel, ERA5","CC-BY-4.0","100K--1M",,"128x128 or 64x64",10,MSI
 `NASA Marine Debris`_,OD,PlanetScope,"Apache-2.0",707,1,256x256,3,RGB
-`OSCD`_,CD,Sentinel-2,"CC-BY-4.0",24,2,"241--1,180",60,MSI
+`OSCD`_,CD,Sentinel-2,"CC-BY-NC-SA",24,2,"241--1,180",60,MSI
```

Comment:
I'm guessing it's the newest one, let's add 4.0 so it's a valid SPDX id: https://spdx.org/licenses/
```suggestion
`OSCD`_,CD,Sentinel-2,"CC-BY-NC-SA-4.0",24,2,"241--1,180",60,MSI
```